CODEOWNERS

```diff
@@ -256,9 +256,9 @@ extensions/filters/http/oauth2 @derekargueta @mattklein123
 /*/extensions/filters/http/rbac @yangminzhu @yanavlasov
 /*/extensions/filters/common/rbac @yangminzhu @yanavlasov
 # tap
-/*/extensions/filters/http/tap @mattklein123 @xu1zhou
-/*/extensions/common/tap @mattklein123 @xu1zhou
-/*/extensions/transport_sockets/tap @mattklein123 @xu1zhou
+/*/extensions/filters/http/tap @mattklein123 @coolg92003 @xu1zhou
```

Comment:
I'm assuming the new tap_sinks extensions directory should also be part of this file.

Reply:
yes, added